mava/evaluator.py

```diff
@@ -311,25 +307,12 @@ def evaluator_setup(
             10,
         )
     else:
-        vmapped_eval_apply_fn = jax.vmap(
-            network.apply,
-            in_axes=(None, 0),
-        )
-        evaluator = get_ff_evaluator_fn(eval_env, vmapped_eval_apply_fn, config, log_win_rate)
+        evaluator = get_ff_evaluator_fn(eval_env, network.apply, config, log_win_rate)
```

Comment:
I really like moving to network.apply and not having to vmap here. I just want us to be super sure that the networks are behaving as expected, ie. broadcasting over the agents. 
Did you running any performance tests on this @sash-a?

Reply:
Nice point @RuanJohn , It may behave the same (@sash-a if you can test it and make sure this is true) and if this is the case something really important to check is steps per seconds

Reply:
I remember testing this ages ago and it was the same, also I do this in SAC and it works fine. Although I'm actually a bit worried about it behaving incorrectly with CNNs 🤔

Reply:
I think a few simple ppo tests would be great for this if possible just to be sure please :pray:

Reply:
Testing...